Hurd/View/Onboarding/OnboardingProfileInfoView.swift

```diff
@@ -78,10 +81,15 @@ struct OnboardingProfileInfoView: View {
                 }
             }
             
-    
             TextField("Phone Number", text: $vm.phoneNumber)
                 .keyboardType(.phonePad)
                 .textFieldStyle(.roundedBorder)
+                .onReceive(vm.$phoneNumber) { newValue in
```

Comment:
?. based on your video, it only does the formatting after the cell is tapped out of. not completely sure. but wondering if we converted this modifier to an ```.onChange```. would each character format until we get the full formatting.